File: Frontend/src/Student.js
import React, { useState } from "react";
import axios from "axios";
function Student() {
	const [name, setName] = useState("");
	const [cnum, setCnum] = useState();
	const [marks, setMarks] = useState();
	const getStudentData = () => {
		axios
			.get("http://localhost:8095/studentDetails")
			.then((response) => {
				setName(response.data[0].name);
				setCnum(response.data[0].contact);
				setMarks(response.data[0].marks);
			})
			.catch((error) => console.log(error));
	};
	getStudentData();
	return (
		<div>
			<table>
				<tr>
					<th>Student name</th>
					<th>Contact number</th>
					<th>Marks</th>
				</tr>
				<tr>
					<td>{name}</td>
					<td>{cnum}</td>
					<td>{marks}</td>
				</tr>
			</table>
		</div>
	);
}

export default Student;

File: example/src/exam/Student.java
package exam;

public class Student {

	public Student(String name, String clas, int marks) {
		super();
		this.setName(name);
		this.clas = clas;
		this.setMarks(marks);
	}
	public int getMarks() {
		return marks;
	}
	public void setMarks(int marks) {
		this.marks = marks;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	private String name;
	private String clas;
	private int marks;
}
